hide comments for non-members

michaelbeil opened the pull request:
this is from @ipokkel - https://gist.github.com/ipokkel/c5dd3b0fe720b3a26aab902928abd224

michaelbeil committed:
hide comments for non-members

michaelbeil committed:
author

michaelbeil committed:
update name

dparker1005 commented on misc/hide-comments-for-non-members.php:
```diff
@@ -0,0 +1,37 @@
+ 	$pmpro_comments_non_member_text = pmpro_get_no_access_message( null, null, null );
+ 	$pmpro_comments_non_member_text = str_replace( __( 'This content is', 'pmpro-comments' ), __( 'Comments are', 'pmpro-comments' ), $pmpro_comments_non_member_text );
+
+ 	echo wp_kses_post( $pmpro_comments_non_member_text );
```
```suggestion
 	echo wp_kses_post( $pmpro_comments_non_member_text );
 }
```

dparker1005 reviewed (commented):
Adding closing bracket

dparker1005 committed:
Update misc/hide-comments-for-non-members.php

dparker1005 commented:
This code does not hide the comment form for non-members, only shows a "no access" message before it. Should we be hiding the comment form as well?

michaelbeil commented:
yes, it should be hiding the comments. this was taken from [trello](https://trello.com/c/WPiOlpdv/358-hide-comments-on-public-posts) for off-support-work (untested, as i thought it did exactly that).

michaelbeil committed:
add intval

I saw @dwanjuki update this so I wanted to make sure it was updated - https://gist.github.com/dwanjuki/77c47dc61489350a8ebe2d0ef18cf931

michaelbeil committed:
Merge branch 'strangerstudios:dev' into dev

michaelbeil committed:
Revert "add intval"

This reverts commit 5c25e5e7a9db443125917f78105576d77476c81f.

michaelbeil committed:
Revert "Merge branch 'strangerstudios:dev' into dev"

This reverts commit 3d971ee1168f4ea3fa5cc0c02efc4b68b3bb3853, reversing
changes made to 5c25e5e7a9db443125917f78105576d77476c81f.

andrewlimaza commented:
I had to redo this due to a bunch of merge conflicts. I refactored it slightly.
